ci-operator/step-registry/openshift-qe/installer/bm/deploy/openshift-qe-installer-bm-deploy-commands.sh

```diff
@@ -26,6 +26,7 @@ ocp_version: $OCP_VERSION
 ocp_build: $OCP_BUILD
 networktype: OVNKubernetes
 public_vlan: $PUBLIC_VLAN
+sno_use_lab_dhcp: false
```

Comment:
It's is going to be always false for all scenarios (mno, sno, public, etc.) ?

Reply:
Basically I have added a var `sno_use_lab_dhcp` which is to trigger jetlag to set SNOs to have zero static networking setup, which ends up deploying the SNO on the lab's DHCP public interface. We don't have any scenarios to use this networking setup in CPT/CI (yet) but since this var is defined in the all.sample.yml, it must be defined here. Alternatively I can have it defined a 2nd time within defaults/main.yml vars file for roles in which the var is referenced however then I duplicate more instances of the var existing. (That is a possibility too)

Reply:
understood!